cairo/src/font/font_face.rs

```diff
@@ -55,7 +55,11 @@ impl FontFace {
         Ok(font_face)
     }
 
-    // Safety: the FT_Face must be valid and not be freed until the `FontFace` is dropped.
+    // rustdoc-stripper-ignore-next
+    /// Creates a new font face for the FreeType backend from an already opened FreeType face.
+    ///
+    /// Safety: the FT_Face must be valid and not be freed until the last reference to the
```

Comment:
While you are doing this, i will keep those consistent. 
```suggestion
    /// # Safety
    /// The FT_Face must be valid and not be freed until the last reference to the
```